cogito/protocol.go

```diff
@@ -9,6 +9,10 @@ import (
 	"strings"
 )
 
+// DummyVersion is the version always returned by the Cogito resource.
+// DO NOT REASSIGN!
+var DummyVersion = Version{Ref: "dummy"}
```

Comment:
We would like this to be a const but it cannot :-/

Reply:
I can, but I should not (code will compile fine; linter will complain). Still, point taken :-/

Reply:
For my understanding: why it cannot be a constant?

Reply:
Because the compiler, implementing the Go language specs, is not able to.
Making things constant (or immutable) is not as easy as it seems from a purely conceptual point of view.
See for example (although confusing) https://github.com/golang/go/issues/27975.
I wish that Go 1 had the concept of immutable, but now it is too late.